integration_tests/tests/integration/common.rs

```diff
@@ -182,8 +236,8 @@ pub async fn submit_measurements_and_verify_aggregate_generic<V>(
     .with_collect_poll_backoff(
         ExponentialBackoffBuilder::new()
             .with_initial_interval(time::Duration::from_millis(500))
-            .with_max_interval(time::Duration::from_millis(500))
-            .with_max_elapsed_time(Some(time::Duration::from_secs(60)))
+            .with_max_interval(time::Duration::from_secs(30))
+            .with_max_elapsed_time(Some(time::Duration::from_secs(600)))
```

Comment:
Shouldn't this use `task_parameters.max_interval` and `task_parameters.max_elapsed_time`?

Reply:
Yes, it absolutely should! Thanks.